README.md

```diff
@@ -0,0 +1,207 @@
+# NGSA Loderunner - A web request validation tool
+
+Loderunner (L8r) is an internal web request validation tool that we use to run end-to-end tests and long-running smoke tests.
+
+## Quick Start
+
+L8r can be run as a docker container, since it is available at gitcr.io, most of its features and functionality are inherited from its parent code base [WebV](https://github.com/microsoft/webvalidate).
+
+## Running as a docker container
+
+```bash
+
+# pull image from GitHub Container Repository 
+docker pull ghcr.io/retaildevcrews/ngsa-lr:beta
+
+```
+
+Experiment with L8r
+
+Run L8r with --help option, this should output command line options shown [below](#command-line-parameters)
+
+```bash
+docker run ghcr.io/retaildevcrews/ngsa-lr:beta --help
+
+```
+
+Run benchmark and baseline test files
+
+```bash
+
+# Replace domainname sever name with actual ngsa-app url 
+docker run --rm ghcr.io/retaildevcrews/ngsa-lr:beta -s "https://DomainName" -f benchmark.json baseline.json
+
+```
+
+Loderunner uses both environment variables as well as command line options for configuration. Command flags take precedence over environment variables.
+
+Loderunner works in two distinct modes. The default mode processes the input file(s) in sequential order one time and exits. The "run loop" mode runs in a continuous loop until stopped or for the specified duration. Some environment variables and command flags are only valid if run loop is specified and L8r will exit and display usage information. Some parameters have different default values depending on the mode of execution.
+
+### Example Arguments for Long Running Tests
+
+```bash
+# continuously send request every 15 seconds
+
+docker run --rm ghcr.io/retaildevcrews/ngsa-lr:beta --sleep 15000 --run-loop --server "https://DomainName" --files benchmark.json
+```
```

Comment:
Since this is new doc for Loderunner, have all of the docker run commands been tested?

Reply:
yes I did, and also removed documentation for args that are not implemented for loderunner